### DISCUSSION TITLE
[Deprecated] Possible Virtualbox install locations

### DISCUSSION THREAD

**Discussion by BhaskarPanja93** [BhaskarPanja93/View-Bot-Client#12] [Q&A]
Please reply (if any other possible location)
Currently working VBoxManage.exe paths:

`C://Program Files/Oracle/VirtualBox/VBoxManage.exe`
`D://Programas/Virtual Box/VBoxManage.exe`

**Comment by Dec1lent** [comment]
`C:\Program Files\Oracle\VirtualBox`, for some reason VirtualBox doesnt add VboxManage.exe to path on install.

**Reply by BhaskarPanja93** [reply]
`C:\Program Files\Oracle\VirtualBox` is added by default to the program, (Maybe try updating the user main exe code from here https://github.com/BhaskarPanja93/Adfly-View-Bot-Client/releases)
If it still doesnt work, open a new issue, and i'll check and fix it

**Reply by BhaskarPanja93** [reply]
> `C:\Program Files\Oracle\VirtualBox`, for some reason VirtualBox doesnt add VboxManage.exe to path on install.

Also i think i needed to mention this, but you dont need the binary to be added in PATH, my programs searches it in default location and uses the file!!
